README.md

```diff
@@ -101,12 +104,12 @@ Manage > Access (IAM) > Access groups > Access policies.
 You need the following permissions to run this module.
 
 - Account Management
-    - **Sample Account Service** service
-        - `Editor` platform access
-        - `Manager` service access
+- **Sample Account Service** service
+- `Editor` platform access
+- `Manager` service access
 - IAM Services
-    - **Sample Cloud Service** service
-        - `Administrator` platform access
+- **Sample Cloud Service** service
+- `Administrator` platform access
```

Comment:
Somehow we need to preserve the indentation so that contributors can copy the list with nested items.